starboard/elf_loader/exported_symbols.cc

```diff
@@ -303,13 +303,11 @@ ExportedSymbols::ExportedSymbols() {
   REGISTER_SYMBOL(SbStorageOpenRecord);
   REGISTER_SYMBOL(SbStorageReadRecord);
   REGISTER_SYMBOL(SbStorageWriteRecord);
+#if SB_API_VERSION < 16
   REGISTER_SYMBOL(SbStringCompareNoCase);
```

Comment:
Are we not going to now register strcasecmp and strncasecmp?

Reply:
We don't need them registered as we were able to use the musl free standing implementation. Take a look at the changes in third_party/musl/BUILD.gn.